.github/workflows/int_test.yml

```diff
@@ -41,38 +41,58 @@ permissions:
   contents: 'read'
   id-token: 'write'
 
+env:
+  KV_NAME: "opskv${{ github.run_number }}"
+  RESOURCE_GROUP: "${{ inputs.resource_group }}"
+
 jobs:
-  integration_test:
+  create_kv:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: "Az CLI login"
+      uses: azure/login@v1
+      with:
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
+        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+    - name: "Create Key Vault for clusters"
+      run: |
+        az keyvault create -n ${{ env.KV_NAME }} -g ${{ env.RESOURCE_GROUP }}
```

Comment:
we should explicitly include `--enable-rbac-authorization false` because its going to default to `true` soon.